cmd/pbstatuser/main.go

```diff
@@ -41,7 +43,29 @@ type SourceInfo struct {
 	Identity identity.XRHID
 }
 
+const (
+	metricNameSourceAvailabilityCheckRequestTotal    = "source_availability_check_request_total"
+	metricNameSourceAvailabilityCheckRequestDuration = "source_availability_check_request_duration"
+)
+
 var (
+	totalAvailabilityCheckReqs = prometheus.NewCounterVec(
+		prometheus.CounterOpts{
+			Name:        metricNameSourceAvailabilityCheckRequestTotal,
+			Help:        "availability Check requests count partitioned by type (aws/gcp/azure), source status and error",
+			ConstLabels: prometheus.Labels{"service": version.PrometheusLabelName},
+		},
+		[]string{"type", "status", "error"},
+	)
+
+	availabilityCheckReqsDuration = prometheus.NewHistogramVec(prometheus.HistogramOpts{
+		Name:        metricNameSourceAvailabilityCheckRequestDuration,
+		Help:        "Availability check Request duration partitioned by type and error",
+		ConstLabels: prometheus.Labels{"service": version.PrometheusLabelName},
+	},
+		[]string{"type", "error"},
+	)
```

Comment:
I have the same comment as in the other PR - it is a good idea to introduce a new package called "metrics" with exported functions for each metrics which allows us to "hide" the actual implementation and protect our app code from these metric definitions.